exporter/logzioexporter/go.mod

```diff
@@ -4,16 +4,12 @@ go 1.16
 
 require (
 	github.com/census-instrumentation/opencensus-proto v0.3.0
-	github.com/hashicorp/go-hclog v0.16.1
+	github.com/hashicorp/go-hclog v0.16.2
 	github.com/jaegertracing/jaeger v1.24.0
-	github.com/logzio/jaeger-logzio v1.0.2
-	github.com/mattn/go-colorable v0.1.7 // indirect
-	github.com/onsi/ginkgo v1.14.1 // indirect
-	github.com/onsi/gomega v1.10.2 // indirect
+	github.com/logzio/jaeger-logzio v1.0.3
 	github.com/stretchr/testify v1.7.0
 	go.opentelemetry.io/collector v0.31.0
 	go.opentelemetry.io/collector/model v0.31.0
 	go.uber.org/zap v1.18.1
 	google.golang.org/protobuf v1.27.1
-	gopkg.in/square/go-jose.v2 v2.5.1 // indirect
```

Comment:
How are these changes related to queue management or solving the linked issue? If they are not please either document it on the PR description or if possible split into separate PR

Reply:
@mx-psi in this version of `jaeger-logzio` we implemented options to use in memory queue before that traces were stored on the fs which caused permission issue (#3736 and #3702) when users deploy the collector in a container because the default container user cannot write to the virtual fs ([PR](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/2925/files))

Reply:
these changes are already were merged in a previous PR (#4386) see [file](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/exporter/logzioexporter/go.mod)

Reply:
About queue management, the update in `jaeger-logzio` made it possible to add the new configuration options:(`DrainInterval`, `QueueCapacity `, `QueueMaxLength`)

Reply:
I see, makes sense. Thanks for explaining